revproxy/response.py

```diff
@@ -4,16 +4,30 @@
 
 from django.http import HttpResponse, StreamingHttpResponse
 
+#: Headers used only for HOP-BY-HOP transport
 HOP_BY_HOP_HEADERS = (
     'connection', 'keep-alive', 'proxy-authenticate', 'proxy-authorization',
     'te', 'trailers', 'transfer-encoding', 'upgrade')
 
+#: Headers that must be ignored
 IGNORE_HEADERS = HOP_BY_HOP_HEADERS + ('set-cookie', )
 
-DEFAULT_AMT = 2 ** 16  # 65536 bytes
+#: Default number of bytes that are going to be read in a file lecture
+DEFAULT_AMT = 2 ** 16
 
 
 def get_django_response(proxy_response):
```

Comment:
Actually this method returns a django http response based on the proxy response. This comment just explains the first if.